frontend/packages/integration-tests-cypress/tests/pseudotranslation.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { checkErrors, testName } from '../support';
+import { nav } from '../views/nav';
+
+describe('Localization', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it('pseudolocalizes navigation', () => {
+    cy.log('test navigation');
+    cy.visit('/dashboards?pseudolocalization=true');
+    nav.sidenav.switcher.shouldHaveText('[Ḥṓṓṃḛḛ]');
```

Comment:
I'd suggest we add a helper that checks if the string starts with `[`, ends with `]`, and doesn't have `a-zA-Z`. Then we can have a list of URLs / CSS selectors to check and add tests really easily without needing to hard-code every pseudo-localization.

Reply:
Hi, it's not `nav.sidenav.switcher`, rather `nav.sidenav.shouldHaveNavSection`.  The `switcher` is 'Administration' .vs 'Developer' views.